### DISCUSSION TITLE
Three quick questions about installing Mecha

### DISCUSSION THREAD

**Discussion by desbest** [mecha-cms/mecha#292] [General]
1. Why does downloading the "release" fail to work and give me an Error 403 Not Authorised but downloading from the `mecha-cms` website however works?
2. Why is the "minify php" checkbox ticked by default. I use PHP 7. Also minifying PHP has no effect on speed. Is that a troll?
3. Why are you using Google for cookie management, of all companies? [Iubenda](https://www.iubenda.com/en/) is a much better solution.

> Frequently Asked Questions
> Do you have any tool to minify PHP code?
> PHP code works on the server side so your clients will never download any PHP code from your server. Only the compiled results will be sent to the client. Minifying PHP code will not improve the performance of your site in any way.
> [[source]](https://minify.js.org/)

**Comment by taufik-nurrohman** [comment]
1. Because the core CMS repository contains sub-modules which will not be automatically downloaded (this is due to the lack of the Git system itself that even GitHub cannot do anything about it to keep the service in sync with the original Git work-flow). The website uses GitHub API to parse the `.gitmodules` file to automatically download the modules as well and then merge them into the core CMS package.
2. I assumed people would prefer that way. Would consider to un-check the PHP minification feature by default then.
3. It may be from the Google AdSense. I have no control over that.
